recipe/meta.yaml

```diff
@@ -22,32 +20,35 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3.5
     - pip
   run:
-    - python
-    - six >=1.5
+    - python >=3.5
+    - sphinxcontrib-applehelp
+    - sphinxcontrib-devhelp
+    - sphinxcontrib-jsmath
+    - sphinxcontrib-htmlhelp
+    - sphinxcontrib-serializinghtml
+    - sphinxcontrib-qthelp
```

Comment:
Not sure we have all these.

Reply:
I'm getting build failures on an upstream project due to missing sphinxcontrib, had to pin my CI to `sphinx<2`.

Reply:
BTW, it should be OK to use 2.0 now that the circluar deps are built.